client/src/pages/AddBookPage.jsx

```diff
@@ -0,0 +1,26 @@
+import { useQuery } from 'react-query'
+import { ReactQueryDevtoolsPanel } from 'react-query/devtools'
+
+const fetchPlanets = async () => {
+  const res = await fetch('http://swapi.dev/api/planets/')
+  return res.json()
+}
```

Comment:
Is this a placeholder API for the other one? If you do a fetch from the same API in two files, you might want to consider fetching in the `App.js` and then pass the result as props.

Reply:
Yes, I was just learning react query 😅